include/experimental/group/gemm/impl/int4_dequantize_xmx_xe.hpp

```diff
@@ -407,6 +415,19 @@ class gemm_t<compute_policy_int4_dequantize_xmx<compute_attr_,
             }
             subgroup::tile_load<cache_hint::cached, cache_hint::cached>(
                     matA, matA_payload);
+            //             sycl::ext::oneapi::experimental::printf("Mat A load :\n ");
```

Comment:
i guess we can make these debug codes as a debug func?

Reply:
yeah